### PR TITLE
added rtcp-fb payload type parameters field suppoty to rest protocol

### PR DESCRIPTION
**Pull request by sidorovis**
according with next issue
https://github.com/jitsi/jitsi-videobridge/issues/568

added possibility to serialize / de-serialize rtcp-fb parameters for REST (colibri) protocol.

### CODE REVIEW

**Comment by bgrozev** [comment]
Jenkins ok to test

File: src/main/java/org/jitsi/videobridge/rest/JSONDeserializer.java [hunk 1]
```diff
@@ -576,6 +576,26 @@ public static void deserializeParameters(
         }
     }
 
+    public static void deserializeRtcpFbs(
+            JSONArray rtcpFbs,
+            PayloadTypePacketExtension payloadTypeIQ) {
+        if (rtcpFbs != null && rtcpFbs instanceof JSONArray) {
```
Comment (bgrozev) [inline src/main/java/org/jitsi/videobridge/rest/JSONDeserializer.java]: rtcpFbs's declared type is JSONArray, there's no need for the instanceof check.

**Comment by bgrozev** [comment]
LGTM functionally. Can you please change the formatting to adhere to our code convention: star-imports only (for NutNull), lines should not exceed 80 characters, opening braces must be on new lines.

Thanks!

**Comment by sidorovis** [comment]
I'm not sure how to pass tests because it shows to me 404 Not Found when I try to get Details for last commit tests.

**Comment by damencho** [comment]
There is a problem with the ci machine. I'm fixing some of the stuff now. Will restart tests and will write here when it is ok.

**Comment by damencho** [comment]
Its working and green :)

**Pull request merged**